superset/config.py

```diff
@@ -946,7 +946,7 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 WEBDRIVER_TYPE = "firefox"
 
 # Window size - this will impact the rendering of the data
-WEBDRIVER_WINDOW = {"dashboard": (1600, 2000), "slice": (3000, 1200)}
+WEBDRIVER_WINDOW = {"dashboard": (1600, 3000), "slice": (1280, 960)}
```

Comment:
This is probably a breaking change, why do we need it?

Reply:
In my tests too big slice resolution makes pictures blur too much. Here is result for 3000x1200 and 1280x960
![image](https://user-images.githubusercontent.com/3406152/108336911-9703f300-71e5-11eb-8e8d-8f59ec777e74.png)
![image](https://user-images.githubusercontent.com/3406152/108337108-d16d9000-71e5-11eb-962a-9fe2497c62e4.png)

Reply:
@dpgaspar For the record. There will be no problem with reports because I provided same thumb_size and window_size for reports. But they still get so blur for thumbnails. I can revert this change if you want.

Reply:
I decided to revert that and provide old harcoded sizes for thumbnails so that caches will not be needed to calculated again.

Reply:
nice, better to change your PR description: `Additional note: This PR will cause thumbnails recalculate because cache keys depends on window size.
`

Reply:
Removed that part